great_expectations/data_context/types/resource_identifiers.py

```diff
@@ -229,6 +229,16 @@ def cloud_id(self):
     def cloud_id(self, value) -> None:
         self._cloud_id = value
 
+    @property
+    def ge_cloud_id(self):
+        # Deprecated 0.15.40
+        return self.cloud_id
+
+    @ge_cloud_id.setter
+    def ge_cloud_id(self, value) -> None:
+        # Deprecated 0.15.40
+        self.cloud_id = value
```

Comment:
Adding these back in to protect against future issues (until we determine a thorough testing and deprecation strategy)

Reply:
🙇🏻